metadata/src/main/java/org/apache/kafka/metadata/storage/Formatter.java

```diff
@@ -348,15 +348,22 @@ private short effectiveKRaftFeatureLevel(Optional<Short> configuredKRaftVersionL
         if (configuredKRaftVersionLevel.isPresent()) {
             if (configuredKRaftVersionLevel.get() == 0) {
                 if (hasDynamicQuorum()) {
-                    throw new FormatterException("Cannot set kraft.version to " +
-                        configuredKRaftVersionLevel.get() + " if KIP-853 configuration is present. " +
-                            "Try removing the --feature flag for kraft.version.");
+                    throw new FormatterException(
```

Comment:
Thanks for the PR. I guess this error only occurs for Kraft.version 0 and the dynamic controller flags are set. Would it be useful to say, "if you would like a dynamic quorum, try removing the ..."

Reply:
Done.